api/types/events/events.proto

```diff
@@ -1319,7 +1319,7 @@ message OneOf {
         events.ClientDisconnect ClientDisconnect = 21;
         events.AuthAttempt AuthAttempt = 22;
         events.AccessRequestCreate AccessRequestCreate = 23;
-        events.ResetPasswordTokenCreate ResetPasswordTokenCreate = 24;
+        events.UserTokenCreate UserTokenCreate = 24;
```

Comment:
This is the one proto change that left me a bit uneasy - if an old version of this was marshaled we'd break trying to read it. It might be safer to deprecate the old `ResetPasswordTokenCreate` and add a new field for `UserTokenCreate`.

I'm not sure what the blast radius of this proto might be, so I'll leave it up to the author and other reviewers.

Reply:
I **don't** believe this is a breaking change.
The only case where it would be breaking is if `OneOf` got serialized to JSON and back. Protobuf serialization should only care about field numbers.

From what I can tell, we store individual event messages in the audit log backend (like `UserLogin`), not the `OneOf` wrapper.

It should be easy to test though - trigger this event on an older version of teleport and then try to view it in the UI on the newer version of teleport.

Reply:
> It should be easy to test though - trigger this event on an older version of teleport and then try to view it in the UI on the newer version of teleport.

tested this with audit log screen and no errors were produced

Reply:
It was the chance that it might be serialized as JSON that I was concerned about, but it sounds like we're good.